libyul/AsmDataForward.h

```diff
@@ -49,6 +48,6 @@ struct Block;
 struct TypedName;
 
 using Expression = boost::variant<FunctionalInstruction, FunctionCall, Identifier, Literal>;
```

Comment:
Maybe
```suggestion
using Expression = boost::variant<FunctionCall, Identifier, Literal>;
```
but I'm not sure we are ready for that yet.